moped-database/migrations/1723496266685_add_fdu_columns/up.sql

```diff
@@ -0,0 +1,11 @@
+-- Add generated columns for fund_dept_unit and fund_name
+ALTER TABLE moped_proj_funding
+ADD COLUMN fund_dept_unit text GENERATED ALWAYS AS (CASE WHEN (fund IS null OR dept_unit IS null) THEN null ELSE
+        coalesce(fund ->> 'fund_id', ' ') || ' ' || coalesce(dept_unit ->> 'dept', ' ') || ' ' || coalesce(dept_unit ->> 'unit', ' ')
+END) STORED,
```

Comment:
this looks good. i did a quick scan of all the dept-units we have in the public dataset, and i didn't see a case where the `dept` or `unit` values are null/empty. if there were any empty values, we could end up with some generated values with extra whitespace, and that might start to be an issue if the downstream/PowerBI users are trying to join these records with other data. i think what you have here is a great solution 🚢